### PR TITLE
Add support for deploying Aqua Tenant Manager.

### PR DESCRIPTION
**Pull request by guyyakir**

### CODE REVIEW

**Review by rshmiel — APPROVED** [review]
approved - @KoppulaRajender waiting for your approval

**Review by KoppulaRajender — APPROVED** [review]

**Review by rshmiel — APPROVED** [review]
Tested

**Pull request merged**
